### PR TITLE
Remove unused conditions

### PR DESCRIPTION
**Pull request by alpaca-tc**
related: https://github.com/winebarrel/ridgepole/pull/335

Removed unused conditions for non-supported version.

### CODE REVIEW

File: spec/mysql/migrate/migrate_create_index_spec.rb [hunk 1]
```diff
@@ -142,15 +142,15 @@
       expect(subject.dump).to match_ruby actual_dsl
       expect(delta.script).to match_fuzzy erbh(<<-ERB)
         change_table("clubs", bulk: true) do |t|
-          t.index(["name"], **<%= {:name=>"idx_name", :unique=>true} + cond(5.0, using: :btree) %>)
+          t.index(["name"], **<%= {:name=>"idx_name", :unique=>true} %>)
```
Comment (winebarrel) [inline spec/mysql/migrate/migrate_create_index_spec.rb:145]:
```suggestion
          t.index(["name"], **{:name=>"idx_name", :unique=>true})
```

File: spec/mysql/migrate/migrate_create_index_spec.rb [hunk 2]
```diff
@@ -142,15 +142,15 @@
...
         end
 
         change_table("employee_clubs", bulk: true) do |t|
-          t.index(["emp_no", "club_id"], **<%= {:name=>"idx_emp_no_club_id"} + cond(5.0, using: :btree) %>)
+          t.index(["emp_no", "club_id"], **<%= {:name=>"idx_emp_no_club_id"} %>)
```
Comment (winebarrel) [inline spec/mysql/migrate/migrate_create_index_spec.rb:149]:
```suggestion
          t.index(["emp_no", "club_id"], **{:name=>"idx_emp_no_club_id"})

File: spec/mysql/migrate/migrate_create_index_spec.rb [hunk 3]
```diff
@@ -142,15 +142,15 @@
...
         end
 
         change_table("titles", bulk: true) do |t|
-          t.index(["emp_no"], **<%= {:name=>"emp_no"} + cond(5.0, using: :btree) %>)
+          t.index(["emp_no"], **<%= {:name=>"emp_no"} %>)
```
Comment (winebarrel) [inline spec/mysql/migrate/migrate_create_index_spec.rb:153]:
```suggestion
          t.index(["emp_no"], **{:name=>"emp_no"})
```

File: spec/mysql/migrate/migrate_execute_spec.rb [hunk 4]
```diff
@@ -11,7 +11,7 @@
         create_table "books", <%= i cond('>= 5.1',id: :integer) + {force: :cascad} %>e do |t|
```
Comment (winebarrel) [inline spec/mysql/migrate/migrate_execute_spec.rb:11]:
```suggestion
        create_table "books",  id: :integer, force: :cascade do |t|
```

**Comment by alpaca-tc** [comment]
I have been finished the replacement work 👍 I'm sorry diff is so large 😢

**Review by winebarrel — APPROVED** [review]
Thank you!
I'll merge it just before release v0.9.0.

**Pull request merged**